Jenkinsfile

```diff
@@ -343,6 +350,49 @@ pipeline {
                 githubNotify(context: "${env.GITHUB_CHECK_ITS_NAME}", description: "${env.GITHUB_CHECK_ITS_NAME} ...", status: 'PENDING', targetUrl: "${env.JENKINS_URL}search/?q=${env.ITS_PIPELINE.replaceAll('/','+')}")
               }
             }
+            stage('Benchmarks') {
+              // agent { label 'metal' }
+              // TODO: tools are only installed in this worker for now
+              agent { label 'worker-854309' }
```

Comment:
As soon as all the baremetals got all the required dependencies we can enable the above agent setup